README.md

```diff
@@ -9,7 +9,7 @@ Pathfinder is currently in alpha so expect some rough edges but it is already us
 - access the full Starknet state history
   - includes contract code and storage, and transactions
 - verifies state using Ethereum
-  - calculates the Starknet state's Patricia-Merkle Trie root on a block-by-block basis and confirms it against L1
+  - calculates the Starknet state's Patricia-Merkle Tree root on a block-by-block basis and confirms it against L1
```

Comment:
"Trie" is actually correct as is - [wiki link](https://en.wikipedia.org/wiki/Trie). It does get a bit confusing since it is also a tree data structure so they can be used somewhat interchangeably. You'll see that most ethereum docs also refer to merkle patricia **trie** e.g. [ethereum.org](https://ethereum.org/en/developers/docs/data-structures-and-encoding/patricia-merkle-trie/).